charts/seed-bootstrap/charts/fluent-bit/values.yaml

```diff
@@ -17,3 +17,10 @@ ports:
 additionalFilters: ""
 additionalParsers: ""
 fluentBitConfigurationsOverwrites: {}
+exposedComponents:
+  tagPrefix: "user-exposed"
+  components:
+  - "kube-apiserver"
+  - "kube-controller-manager"
+  - "kube-scheduler"
+  - "cluster-autoscaler"
```

Comment:
"Hard-coding" them here does not fit to #2754. I would suggest to enhance the logging contract in a meaningful way, see https://github.com/gardener/gardener/blob/master/pkg/operation/botanist/component/interfaces.go#L60-L61 and https://github.com/gardener/gardener/blob/master/pkg/operation/botanist/controlplane/kubescheduler/logging.go and https://github.com/gardener/gardener/blob/master/pkg/operation/botanist/controlplane/clusterautoscaler/logging.go.

So far, it's only implemented for kube-scheduler and cluster-autoscaler. Hence, it's fine to keep kube-apiserver and kube-controller-manager for now (although, KCM is being refactored as part of #2881, cc @danielfoehrKn, and KAPI will follow soon).

Reply:
Can you check now, please